graph-wrapper.cabal

```diff
@@ -1,7 +1,7 @@
 Cabal-Version:      >= 1.8
 Build-Type:         Simple
 Name:               graph-wrapper
-Version:            0.2.5.2
+Version:            0.2.6.0
```

Comment:
@gelisam: I think the PVP only forces us to bump up the third component. (Is that correct?) Any reason you would want this to be a more major bump?